python/example.py

```diff
@@ -81,11 +85,12 @@ def parse_arguments():
                         required=False, default=False)
     parser.add_argument('-certs', '--trustedCertificates', type=str,
       help='Path to trusted certificates for encrypted connection', required=False)
+    parser.add_argument('-output', '--output-file', type=str, help='output file', required=False, default='output.csv')
     return parser.parse_args()
 
 
 def connect_to_dremio_flight_server_endpoint(hostname, flightport, username, password, sqlquery,
-  tls, certs, disableServerVerification):
+  tls, certs, disableServerVerificationi, output_file):
```

Comment:
There's an extra "i" after disableServerVerification now.

Reply:
Missed that! thanks, corrected